MatomoTracker.xcodeproj/project.pbxproj

```diff
@@ -471,14 +475,17 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TVOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
According to the [MatomoTracker.podspec](https://github.com/matomo-org/matomo-sdk-ios/blob/develop/MatomoTracker.podspec) we support tvOS >= 9.0. I see no reason why that should be wrong. What do you think about that? Should we change this here to 9.0 and be consistent?

Reply:
Good point, you're right. I will change that on my site to be consistent.